internal/namespaces/instance/v1/custom_server.go

```diff
@@ -480,6 +480,27 @@ func serverRebootCommand() *core.Command {
 	}
 }
 
+func serverWaitCommand() *core.Command {
+	return &core.Command{
+		Short:     `Wait for server to reach a stable state`,
+		Long:      `Wait for server to reach a stable state. This command is equivalent to using the --wait flag when performing an action on a server.`,
+		Namespace: "instance",
+		Resource:  "server",
+		Verb:      "wait",
+		ArgsType:  reflect.TypeOf(instanceActionRequest{}),
+		Run: func(ctx context.Context, argsI interface{}) (i interface{}, err error) {
+			return waitForServerFunc()(ctx, argsI, nil)
```

Comment:
The third argument does not seem to be used anywhere. 

Is it really necessarry to keep it ?